docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,214 @@
+
+```sh
+$ git clone https://github.com/kubernetes/kubernetes.git
+$ cd kubernetes
+$ hack/build-go.sh
+```
+
+Copy the built kube-scheduler binary into the container image. Here is the `Dockerfile` to build the image:
+
+```Dockerfile
+FROM busybox
+ADD _output/local/go/bin/kube-scheduler /usr/local/bin/kube-scheduler
+```
+
+Save the file as `Dockerfile`, build the image and push it to a registry. This example pushes the image to [Google Container Registry (GCR)](https://cloud.google.com/container-registry/). For more details, please read the GCR [documentation](https://cloud.google.com/container-registry/docs/).
+
+```sh
+$ docker build -t my-kube-scheduler:1.0 .
+$ gcloud docker push gcr.io/my-gcp-project/my-kube-scheduler:1.0
+```
+
+### 2. Define a Kubernetes deployment for the scheduler
+
+Like most other system components, Kubernetes allows running a scheduler in a [pod](http://kubernetes.io/docs/user-guide/pods/). Since we have our scheduler in a container image already, we can just create a pod config for it and run it in a Kubernetes cluster. But instead of just running a pod in the cluster, let's use a [Deployment](http://kubernetes.io/docs/user-guide/deployments/) for this example. A [Deployment](http://kubernetes.io/docs/user-guide/deployments/) manages a ReplicaSet which in turn manages the pods, thereby making the scheduler resilient to failures. Here is the deployment config. Save it as `my-scheduler.yaml`:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchExpressions:
+    - {key: tier, operator: In, values: [control-plane]}
+    - {key: component, operator: In, values: [scheduler]}
+    - {key: version, operator: In, values: [second]}
+  template:
+    metadata:
+      labels: 
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command: [/usr/local/bin/kube-scheduler, --address=0.0.0.0,
+                  --scheduler-name=my-scheduler]
+        image: gcr.io/my-gcp-project/my-kube-scheduler:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+```
+
+An important thing to note here is that the name of the scheduler specified as an argument to the scheduler command in the container spec should be unique. This is used to map pods to schedulers.
```

Comment:
s/This is used to map pods to schedulers./This is the name that is matched against the value of the optional `scheduler.alpha.kubernetes.io/name` annotation on pods, to determine whether this scheduler is responsible for scheduling a particular pod./